README.md

```diff
@@ -94,6 +94,17 @@ Array of [rehype plugins](https://mdxjs.com/advanced/plugins#using-remark-and-re
 | `process` | `function` | A hook function whose return value will be appended to the processed front matter. This function is given access to the source `.mdx` content as the first parameter. |
 | `phase`   | `string`   | Used to specify when to run the `process` function. Eligible values are `prebuild`, `loader`, `both`. Defaults to `both` if not specified.                            |
 
+### scan
+
+> `object` | optional
+
+Object of scan objects containing the following parameters
+
+| Property    | Type                | Description                                                                                                                                                                                                                                                                                                                                          |
+| ----------- | ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `pattern`   | `RegEx`             | A RegEx to use for scanning `.mdx` content, enables Layout customization                                                                                                                                                                                                                                                                             |
+| `transform` | `string` _optional_ | An optional callback function that transforms the result of the match operation. This function is passed an Array of any matching `.mdx` content that is returned by [`content.match(pattern)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#Return_value) operation utilizing the `pattern` RegEx. |
```

Comment:
Something like this...up to you if you want to provide any sort of type annotations/details

```
| `transform` | function(match: Array[]): any | optional | ... |
```

Reply:
My bad here, super helpful thanks @pruett !